src/main/java/eu/dissco/core/translator/service/DwcaService.java

```diff
@@ -116,21 +132,28 @@ private void extractACMultimedia(ArchiveFile extension) throws JsonProcessingExc
       if (idTobeIgnored.contains(rec.id())) {
         log.info("Skipped media with id: {} as the specimen has been ignored", rec.id());
       } else {
-        var type = determineDigitalMediaType(extension, rec);
+        var type = termMapper.retrieveFromDWCA(new MediaType(), extension, rec);
         log.info("Type of digitalMediaObject is: {}", type);
         var digitalMediaObject = new DigitalMediaObject(
             type,
             rec.id(),
-            rec.value(extension.getField("ac:accessUri").getTerm()),
-            rec.value(extension.getField(DCTERMS_FORMAT).getTerm()),
-            webClientProperties.getSourceSystemId(), getData(extension, rec, false),
-            getData(extension, rec, true),
-            "DWCA");
+            "DWCA",
+            harmonizeMedia(extension, rec),
+            getOriginalAttributes(extension, rec));
```

Comment:
I would move the ending of such a statement (the ); part) to one line below to complete the indent with the top